server/executor/runner_test.go

```diff
@@ -53,8 +53,6 @@ func TestPersistentRunner(t *testing.T) {
 
 		run2 := f.mockDB.runs[test2.ID.String()]
 		require.NotNil(t, run2)
-
-		assert.True(t, run1.CompletedAt.UnixNano() > run2.CompletedAt.UnixNano(), "test1 did not complete after test2")
```

Comment:
Does this test still makes sense? I feel that having a `runner` test mocking everything doesn't provide great value right now that we have integration tests.